modules/deploy/helms/fluentd-values.erb

```diff
@@ -0,0 +1,9 @@
+elasticsearch:
+  ImageTag: "elasticsearch"
+  prefix: "<%= ENV["TF_VAR_environment"].tr('-.','_') %>_k8s_logs"
```

Comment:
Sorry, but I don't understand this. The **tr** method will substitute either `-` or `.` by a `_` character to avoid confusions in the indexes of the Elasticsearch. The index name will be composed by the prefix + the date. For example: `dev_alf_k8s_logs-2018.02.13`
Perhaps we can use a double `_` to separate the var environment and the string k8s_logs, for example:  `dev_alf__k8s_logs-2018.02.13`, so we can realize of each part of the index.